cmd/gendoc/main.go

```diff
@@ -284,9 +301,15 @@ func execExampleMethod(methodFullPackage, methodPackage, methodName string) (str
 		fmt.Print(str)
 	}
 		`, methodFullPackage, methodPackage, methodName))
-	tf.Close()
+	if err != nil {
+		fmt.Print(err)
+	}
+
+	defer tf.Close()
```

Comment:
IMO, closing something that we know for a fact that wont be used anymore shouldn't be a codesmell

Reply:
I guess that we can remove it. Normally GoLang GC should close `os.File` resources by default in this case, when it hits the end of the program.